### PR TITLE
v1.0.0 changelog

### PR DESCRIPTION
**Pull request by sairina**
## Description

Adds a changelog for v1.0.0.

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] The change has been added to the `changelog`

## Reviewer guidance

- [ ] Double check that the links are correctly linked

## Post-merge updates and tracking

- [ ] Learning Platform update PR is submitted
- [ ] Learning Platform update PR is merged
- [ ] Studio update PR is submitted
- [ ] Studio update PR is merged
- [ ] Data Portal update PR is submitted
- [ ] Data Portal update PR is merged

### CODE REVIEW

**Review by indirectlylit — COMMENTED** [review]
This is great, thank you!

After spending some time with it, my main concern with the organization is that the information density is a bit low.

I made a gist linked below which attempts a denser organization by removing some sub-sections, and also addresses some of the other feedback I left inline below. Feel free to use those edits if helpful.

Comparison:
* https://github.com/learningequality/kolibri-design-system/blob/f3354928d94f84c12a8bc6becb680e89a7331868/CHANGELOG.md
* https://gist.github.com/indirectlylit/072f534c8ec186573df302a9b03dfc67

image

The last step will be removing references to the old changelog page:

image

**Comment by indirectlylit** [comment]
One other minor note: [Prettier](https://prettier.io/) supports Markdown! Personally I have it run automatically in my text editor which keeps things like whitespace and syntax consistent over time.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+- #[229] - Add `warningIncomplete` and rename `publishedResource` to `unpublishedResource`
+
+
+__Changed__
+- #[192] - Update design principles wording
+- #[144] - Update page headers
+- #[186] - Update `KCheckbox` and `KSwitch` examples
+
+<!-- Referenced PRs -->
+[153]: https://github.com/learningequality/kolibri-design-system/pull/153
```
[the remark this answers is not in the document]
Comment (sairina) [reply CHANGELOG.md:123]: Ah! That works?! I was using the markdown preview option in VSCode and it seemed like it wouldn't show up, so I didn't do it. But - yes, I agree. I'll make this change.

[the remark this answers is not in the document]
Comment (sairina) [reply CHANGELOG.md:123]: Good call! Of course that works - I made a mistake and then thought it wasn't working. I'll update that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+# 0.2.1
+
+## Content updates
+___
+- Added design principles documentation - #[95]
+- Added errors documentation - #[97]
+- Added switches documentation - #[105]
+- Added menu documentation - #[106]
+
```
[the remark this answers is not in the document]
Comment (sairina) [reply CHANGELOG.md:174]: I kept it from the original changelog, but it makes sense to have it all be the same moving forward!

**Pull request merged**